ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -12,7 +12,24 @@ const TWO_POINTS = 6;
 	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
 	typeof define === 'function' && define.amd ? define(['exports'], factory) :
 	(global = global || self, factory(global.IBRSDK = {}));
-}(this, (function (exports) { 'use strict';
+}(this, (function (exports) {
+    'use strict';
+
+    /**
+     * Decode raw ibr data into structures.
+     * @param {Byte} data Binary data read directly from .ibr file
+     * @return {Array.<Object>} structures List of structure objects generated from input ibr data
+     */
+    function renderStructure(data) {
+        var deserializedData = InternalBuildingRepresentation.read(new Pbf(data));
+        var structures = [];
+        if ( deserializedData.structures.length > 0 ) {
+            structures = deserializedData.structures;
+        } else {
+            structures.push( deserializedData );
```

Comment:
this seems like it would return the whole ibr data if there's no structures, is that intended?

just a note, currently the building IBR that contains the four floors IBR has no data on it, but eventually it could/would. So this if/else would drop that data since it looks like it only does the outer layer or the inner structures but not both.

Reply:
pushed top level structure to deserializedData along with possible children structures.